manifests/s/SomePythonThings/WingetUIStore/0.4/SomePythonThings.WingetUIStore.installer.yaml

```diff
@@ -0,0 +1,22 @@
+# Created with YamlCreate.ps1 v2.0.0 $debug=QUSU.7-1-4
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: SomePythonThings.WingetUIStore
+PackageVersion: "0.4"
+InstallerLocale: en-US
+MinimumOSVersion: 10.0.0.0
+InstallerType: inno
+Scope: user
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+InstallerSwitches:
+  InstallLocation: /DIR="<INSTALLPATH>"
```

Comment:
```suggestion
```
Are these needed? Since it's already included by default when the `InstallerType` is set to `Inno`:
https://github.com/microsoft/winget-cli/blob/2a9d1c66a32af0f44f6dd9b6ad76e88d427efc11/src/AppInstallerCommonCore/Manifest/ManifestCommon.cpp#L470-L477